src/Momento.Sdk/Config/Middleware/LoggingMiddleware.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Grpc.Core.Interceptors;
+
+namespace Momento.Sdk.Config.Middleware
+{
+    public class LoggingMiddleware : IMiddleware
```

Comment:
This is just an example; we may delete it when we have better examples.  For as long as we keep it around we should switch it over from Console.WriteLine to use the logger, as soon as Michael lands that PR.